layouts/index.html

```diff
@@ -6,7 +6,7 @@
 {{ partial "header-front" . }}
 
 <div class="main-content home-page">
-  <p class="intro">Secure your email server with STARTTLS Everywhere!  Your email service can be insecure in <a href="/faq#insecure">numerous</a> <a href="https://zakird.com/papers/mail.pdf" target="_blank">different</a> <a href="https://www.eff.org/deeplinks/2014/12/email-encryption-surged-2014-still-needs-work-2014-review" target="_blank">ways</a>. The service below performs a quick check of your email server's security configuration, including whether STARTTLS is supported, and whether it may qualify for the <a href="/policy-list">STARTTLS Policy List</a>. </p>
+  <p class="intro">Secure your email server with STARTTLS Everywhere!  Your email service can be insecure in <a href="/faq#insecure">numerous</a> <a href="https://zakird.com/papers/mail.pdf" target="_blank">different</a> <a href="https://www.eff.org/deeplinks/2014/12/email-encryption-surged-2014-still-needs-work-2014-review" target="_blank">ways</a>. The service below performs a quick check of your email server's security configuration, including whether STARTTLS and <a href="/faq#mtasts">MTA-STS</a> are supported, and whether it may qualify for the <a href="/policy-list">STARTTLS Policy List</a>. </p>
```

Comment:
I think I remember some style guide from a long time ago that suggests not making successive words in a sentence into separate links to parallel resources (although I know it's a common strategy). This had to do with a goal of trying to help users anticipate what they're going to find at the end of a link, where possible. Is there a way that we could do that here?